feat(docs): Describe coordinate systems and reference points (#510) (#516)
OpenSimulationInterface/open-simulation-interface: doc/usecases/transforming_coordinate_systems.adoc

= Transforming coordinate systems

Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.

> //TODO: Should we add one or more sentences about the mathematical operations involved?

Reviewer: In my opinion that would be very helpful. Also pictures help here a lot, I think.

**Corresponding messages**

``GroundTruth::moving_object::base::position``::
Defines the position of the vehicle’s reference point.

``GroundTruth::moving_object::base::orientation``::
Defines the orientation of the vehicle’s reference point.

``GroundTruth::moving_object::vehicle_attributes::bbcenter_to_rear``::
The vector pointing from the vehicle’s reference point to the middle of the rear axle under neutral load conditions in vehicle coordinates.
The middle of the rear axle is 

``SensorData::mounting_position``::
Defines the sensor’s position and orientation and thereby the origin of the sensor coordinate system.

> The mounting position is defined in sensor coordinates and relative to the origin of the vehicle coordinate system.

Reviewer: ```suggestion
The mounting position is given in vehicle coordinates.
```
